docker-compose.dev.yml

```diff
@@ -0,0 +1,33 @@
+services:
+  app:
+    depends_on:
+      - mongodb
+    build: .
+    restart: unless-stopped
+    environment:
+      - DISCORD_TOKEN=YOUR TOKEN
+      - DISCORD_CLIENT_ID=YOUR CLIENT ID
+      - DISCORD_GUILD_ID=YOUR GUILD ID
+      ### LEAVE THIS AS IT IS! ###
```

Comment:
I think it's better to not say anything, since users **SHOULD** change this, as it's insecure to not do it. Maybe adding a .env for the docker compose or something. So you can use variables and make users configure environments in a file. For example using these files it's possible. ``docker-compose.yml`` ``.env``. I don't know.